faq/domains-and-dns.mdx

```diff
@@ -0,0 +1,110 @@
+---
+meta:
+  title: Domains & DNS
```

Comment:
The final product name is « Domains **and** DNS », as the character `&` is not allowed anymore. I'm afraid you'll have to make a search and replace everywhere.